proposals/stage-0/COLLAB_SUMMIT/README.md

```diff
@@ -0,0 +1,48 @@
+# OpenJS Collaborator Summit
+>  Stage 0
+
+## Champion
+
+Matteo Collina (@mcollina)
+
+## Description
+
+Organize an OpenJS Collaborator Summit twice a year, one in Europe and
+one in the US, following the lead of the [Node.js Collaborator Summit](https://github.com/nodejs/summit).
+
+The goal of the Collaborator Summit is to foster collaboration within
+and across projects in the Foundation.
+
+## Required Resources
+
+* Budget for venue
+* Budget for travel fund
+
+## Who would be responsible?
+
+A team/working group chartered under the CPC that includes
+representation from all interested projects and OpenJS staff.
+The team/working group will oversee the budget for the venue and the
+travel fund.
+
+## How would success be measured?
+
+* numbers of attendees at the Summit. The one in Berlin/Europe 2019 has 70 people
+attending.
+* number of projects partecipating in the summit.
+
+## Why this proposal is important
+
+Meeting fellow collaborators in person is important for the success of
+an Open Source projects.
+
+## Unresolved Question
+
+* The Node.js TSC and CommComm jointly approve the Node.js travel fund
+  requests in https://github.com/nodejs/admin. The majority of this
+  budget is currently spent for attending the summit.
```

Comment:
What's the question here?

Reply:
The question is probably: How will it be managed going forward? It was managed by Node.js, but the Collaborator Summit will now be larger than just Node.js. Will Node.js maintain their travel fund and a new travel fund will be set up for the Foundation as a whole? Will many projects get travel budgets? And so on.

The realization that the summit is now for the entire Foundation and not just Node.js also calls out for a rethink on the whole "Europe + North America" thing.